cli/cmd/login.go

```diff
@@ -160,20 +161,21 @@ func runLogin(ctx context.Context, cmd *cobra.Command) (err error) {
 	}
 
 	if cmd.Flags().Changed("team") {
-		team := cmd.Flag("team").Value.String()
-		token, err := auth.GetToken()
+		selectedTeam := cmd.Flag("team").Value.String()
+		tc := auth.NewTokenClient()
+		token, err := tc.GetToken()
 		if err != nil {
-			return fmt.Errorf("failed to get auth token: %w", err)
+			return fmt.Errorf("failed to get team token: %w", err)
```

Comment:
this doesn't seem right; it's still a per-user auth token

Reply:
Whoops, over enthusiastic renaming by Intellij